engine/src/hopeit/app/api.py

```diff
@@ -183,4 +209,10 @@ def event_api(title: Optional[str] = None,
             }
         )
     """
-    return partial(_event_api, title, payload, query_args, responses)
+
+    if title is not None:
+        warnings.warn("title parameter is deprectated, use summary instead", DeprecationWarning)
+        if summary is None:
+            summary = title
```

Comment:
I couldn't find a test case for title being used instead of summary. Needed to guarantee compatibility until we remove support.

Reply:
add engine/test/mock_app/mock_app_api_title.py to engine/test/unit/app/test_app_api.py

Reply:
Ok now.